documentation/docs/01_installation.md

```diff
@@ -1,12 +1,14 @@
-# From release
+# Installation
 
-# From source
+## From release
```

Comment:
Do you want to add instructions here?

Reply:
Yes I will, just waiting for the first 1.0 release to happen (later today)